tools/BuildPackagesTask/map.json

```diff
@@ -0,0 +1,155 @@
+  ],
+  "src/ResourceManager/Scheduler/": [
+    ".\\src\\ResourceManager\\Scheduler\\Commands.Scheduler.Test\\bin\\Debug\\Microsoft.Azure.Commands.Scheduler.Test.dll"
+  ],
+  "src/ResourceManager/ServerManagement/": [],
+  "src/ResourceManager/ServiceBus/": [
+    ".\\src\\ResourceManager\\ServiceBus\\Commands.ServiceBus.Test\\bin\\Debug\\Microsoft.Azure.Commands.ServiceBus.Test.dll"
+  ],
+  "src/ResourceManager/ServiceFabric/": [
+    ".\\src\\ResourceManager\\ServiceFabric\\Commands.ServiceFabric.Test\\bin\\Debug\\Microsoft.Azure.Commands.ServiceFabric.Test.dll"
+  ],
+  "src/ResourceManager/SiteRecovery/": [
+    ".\\src\\ResourceManager\\SiteRecovery\\Commands.SiteRecovery.Test\\bin\\Debug\\Microsoft.Azure.Commands.SiteRecovery.Test.dll"
+  ],
+  "src/ResourceManager/Sql/": [
+    ".\\src\\ResourceManager\\Sql\\Commands.Sql.Test\\bin\\Debug\\Microsoft.Azure.Commands.Sql.Test.dll"
+  ],
+  "src/ResourceManager/Storage/": [
+    ".\\src\\ResourceManager\\Storage\\Commands.Management.Storage.Test\\bin\\Debug\\Microsoft.Azure.Commands.Management.Storage.Test.dll"
+  ],
+  "src/ResourceManager/StreamAnalytics/": [
+    ".\\src\\ResourceManager\\StreamAnalytics\\Commands.StreamAnalytics.Test\\bin\\Debug\\Microsoft.Azure.Commands.StreamAnalytics.Test.dll"
+  ],
+  "src/ResourceManager/Tags/": [],
+  "src/ResourceManager/TrafficManager/": [
+    ".\\src\\ResourceManager\\TrafficManager\\Commands.TrafficManager2.Test\\bin\\Debug\\Microsoft.Azure.Commands.TrafficManager.Test.ScenarioTests.dll"
+  ],
+  "src/ResourceManager/UsageAggregates/": [
+    ".\\src\\ResourceManager\\UsageAggregates\\Commands.UsageAggregates.Test\\bin\\Debug\\Microsoft.Azure.Commands.UsageAggregates.Test.dll"
+  ],
+  "src/ResourceManager/Websites/": [
+    ".\\src\\ResourceManager\\Websites\\Commands.Websites.Test\\bin\\Debug\\Microsoft.Azure.Commands.Websites.Test.dll"
+  ],
+  "src/ServiceManagement/": [
+    ".\\src\\ServiceManagement\\Common\\Commands.Common.Test\\bin\\Debug\\Microsoft.WindowsAzure.Commands.Common.Test.dll",
+    ".\\src\\ServiceManagement\\Services\\Commands.Test\\bin\\Debug\\Microsoft.WindowsAzure.Commands.Test.dll",
+    ".\\src\\ServiceManagement\\StorSimple\\Commands.StorSimple.Test\\bin\\Debug\\Microsoft.WindowsAzure.Commands.StorSimple.Test.dll",
+    ".\\src\\ServiceManagement\\RemoteApp\\Commands.RemoteApp.Test\\bin\\Debug\\Microsoft.Azure.Commands.RemoteApp.Tests.dll",
+    ".\\src\\ServiceManagement\\Common\\Commands.ScenarioTest\\bin\\Debug\\Microsoft.WindowsAzure.Commands.ScenarioTest.dll",
+    ".\\src\\ServiceManagement\\RecoveryServices\\Commands.RecoveryServices.Test\\bin\\Debug\\Microsoft.Azure.Commands.RecoveryServices.Test.dll",
+    ".\\src\\ServiceManagement\\Network\\Commands.Network.Test\\bin\\Debug\\Microsoft.WindowsAzure.Commands.ServiceManagement.Network.Test.dll"
+  ],
+  "src/Storage/": [
+    ".\\src\\Storage\\Commands.Storage.Test\\bin\\Debug\\Microsoft.WindowsAzure.Commands.Storage.Test.dll"
+  ],
```

Comment:
I assume this is the test map.  Should make analogous chanegs as  specified above.